docs/KEYBINDINGS_OSX.md

```diff
@@ -118,6 +118,7 @@ MarkText key bindings for macOS. Please see [general key bindings](KEYBINDINGS.m
 | `view.toggle-toc` .     | <kbd>Command</kbd>+<kbd>K</kbd>                  | Toggle table of contents                 |
 | `view.toggle-dev-tools` | <kbd>Command</kbd>+<kbd>Alt</kbd>+<kbd>I</kbd>   | Toggle developer tools (debug mode only) |
 | `view.dev-reload`       | <kbd>Command</kbd>+<kbd>R</kbd>                  | Reload window (debug mode only)          |
+| `view.reload-images`    | <kbd>Command</kbd>+<kbd>R</kbd>                  | Reload images                            |
```

Comment:
We'll do a full review later on, but this shortcut is bound twice on macOS. Please select another shortcut for `view.dev-reload` that is preferably not often used and edit `src/main/keyboard/keybindingsDarwin.js` too.

Reply:
/cc @Jocs